SemDiff.Core/GitHub.cs

```diff
@@ -51,6 +52,25 @@ public GitHub(string repoOwner, string repoName, string authUsername = null, str
         public HttpClient Client { get; private set; }
         public string RepoFolder { get; set; }
         public string EtagNoChanges { get; set; }
+        public IList<PullRequest> currentSaved { get; set; }
+        public string JsonFileName { get; set; }
+
+
+        private void GetCurrentSaved()
+        {
+            try
+            {
+                var path = RepoFolder.Replace('/', Path.DirectorySeparatorChar);
+                path += Path.DirectorySeparatorChar + JsonFileName;
+                var json = File.ReadAllText(path);
+                currentSaved = JsonConvert.DeserializeObject<IList<PullRequest>>(json);
+            }
+            catch
+            {
+                //Catch is only here to be used if the json file does not exist. 
+                //If it doesn't exist, ignore.
```

Comment:
I would use the Logger here to log the exception, that way we know if something fishy is happening